src/instances.ts

```diff
@@ -205,11 +205,11 @@ function successfulTypeScriptInstance(
   let normalizedFilePath: string;
   try {
     const filesToLoad = loaderOptions.onlyCompileBundledFiles
-      ? configParseResult.fileNames.filter(fileName =>
+      ? configParseResult.fileNames.filter((fileName: string) =>
```

Comment:
is the type annotation necessary?  Presumably the type can be inferred?

Reply:
I got an error in VS code, that's why I fixed it. But now when I check the tsconfig I can't find "noImplicitAny", so I'm not actually sure why I got that warning. VSCode has been a little shaky when it comes to errors since last release though. I don't have a problem removing them.

Reply:
Fixed